### DISCUSSION TITLE
ESLint Warning

### DISCUSSION THREAD

**Discussion by elliewalkersm** [nss-evening-cohort-14/evening-client-side#167] [Rubber Duck]
Ellie Walker

Part 4

Have you searched the other tickets? Yes

Errors:
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):

/Users/lindseywalker/workspace/projects/personal/ASSIGNMENT-Firebase-Pinterest/src/javascripts/events/domEvents.js
  67:11  warning  Unexpected confirm  no-alert

Should I just disable the eslint for this line?

**Comment by drteresavasquez** [comment]
@elliewalkersm do you have a link to the docs to disable it?

**Reply by elliewalkersm** [reply]
https://eslint.org/docs/rules/no-alert

**Reply by elliewalkersm** [reply]
https://flaviocopes.com/how-to-disable-eslint-rule/

**Reply by drteresavasquez** [reply]
I think @ajawashington waked with you through that one but you could either use the ignore or just get rid of the confirm. That was an example for the almost Amazon requirements you all put on the project.
